tutorial/openmp.md

```diff
@@ -0,0 +1,141 @@
+# Pyccel OpenMP usage
+
+## Using the Runtime Library Routines
+
+OpenMP Runtime Library Routines for Pyccel work by importing the OpenMP routine needed from the Pyccel stdlib:
+
+```python
+from pyccel.stdlib.internal.openmp import omp_set_num_threads
+```
+
+#### Example
```

Comment:
This codacy error seems silly. Maybe we should deactivate it in .codacy.yml

Reply:
i agree

Reply:
Should be fixed. I turned off this error check on codacy